_posts/2017/2017-03-28-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "ePages Hackathon 2017"
+date: "2017-03-28 06:39:20"
+image: blog-header/blue-sky.jpg
+categories: events
+authors: ["Rose"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+As every year in spring, last week we organized our third hackathon at ePages.
+This year's hackathon has been all about coding for and further developing our very latest backend architecture ePages BEYOND, adding value to our current core shop software ePages 6 as well as automating internal processes.
+
+### Getting prepared
+
+Just as last year, our Scrum Masters did an outstanding job organizing and ensuring a smooth flow of the two-day hackathon.
+They divided up teams that usually don't work with each other on a regular basis, they agreed upon final topics, booked meeting rooms for the teams and most importantly ordered delicious snacks, drinks and food {% emoji wink %}.
```

Comment:
Everybody signed up for a group themselves so I would rather go for this:

They organized travel tickets and hotels rooms for the colleagues from other locations, checked with our CTO upon final topics and the time schedule, prepared meeting rooms for the teams, and most importantly ordered delicious snacks, drinks, and food {% emoji wink %}.